sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -294,7 +323,7 @@ class PlanResolutionSuite extends AnalysisTest {
         |COMMENT 'This is the staging page view table'
         |LOCATION '/user/external/page_view'
         |TBLPROPERTIES ('p1'='v1', 'p2'='v2')
-        |AS SELECT * FROM src
+        |AS SELECT 1
```

Comment:
Why did this change? To avoid needing to look up `src`?

These test cases originally came directly from the SparkSql parser tests. Not changing them was a way to ensure compatibility.